packages/react-dom/src/events/plugins/ModernEnterLeaveEventPlugin.js

```diff
@@ -165,6 +162,14 @@ const EnterLeaveEventPlugin = {
     enter.target = toNode;
     enter.relatedTarget = fromNode;
 
+    // If we are not processing the first ancestor, then we
+    // should not process the same nativeEvent again, as we
+    // will have already processed it in the first ancestor.
+    const nativeTargetInst = getClosestInstanceFromNode(nativeEventTarget);
```

Comment:
This is basically the same as what we do on:

https://github.com/facebook/react/blob/master/packages/react-dom/src/events/plugins/LegacyEnterLeaveEventPlugin.js#L168-L173